_includes/configuration/pagebreak_separator.md

```diff
@@ -0,0 +1,18 @@
+### `pagebreak_separator`
```

Comment:
Fun... this setting is missing a description of what it does :cry:

Reply:
:disappointed: